commands/cutoff.js

```diff
@@ -1,10 +1,14 @@
+const { sendInfoMessage } = require("../helpers/utils.js");
 const BEGINNING_SEARCH_YEAR = 2008;
 const logger = require ("../logger")("cutoff");
 const getDebugContext = require("../helpers/utils").getDebugContext
+
 module.exports = {
     call: ({ message, parsedMessage, guildPreference, db }) => {
         guildPreference.setBeginningCutoffYear(parseInt(parsedMessage.components[0]), db);
-        message.channel.send(`The new cutoff year is \`${guildPreference.getBeginningCutoffYear()}\`.`);
+        sendInfoMessage(message,
+            "**Cutoff year**",
```

Comment:
Doesn't `sendInfoMessage` already wrap the title in asterisks?

Reply:
Whoops, you're right. Will fix this right now.